src/main/java/com/metamx/collections/bitmap/RoaringBitmapFactory.java

```diff
@@ -58,9 +61,7 @@ public MutableBitmap makeEmptyMutableBitmap()
   @Override
   public ImmutableBitmap makeEmptyImmutableBitmap()
   {
-    return new WrappedImmutableRoaringBitmap(
-        EMPTY_IMMUTABLE_BITMAP
-    );
+    return WRAPPED_IMMUTABLE_ROARING_BITMAP;
```

Comment:
I know its not different from what it was previously, but I don't know if the previous has gotten enough use to say there wasn't a problem.